lib/main.js

```diff
@@ -20,34 +20,17 @@ var notifications = require('sdk/notifications');
 var {setTimeout, clearTimeout} = require('sdk/timers');
 var snooze = require('./snooze');
 var tabs = require('sdk/tabs');
-var {Widget} = require('sdk/widget');
+var {ToggleButton} = require('sdk/ui/button/toggle');
 
 const STUDY_ID = 'snoozetabs';
 const UPLOAD_URL = 'https://snoozetabs.paas.allizom.org/data/' + STUDY_ID;
 
 
 var makeContent = function (image) {
-  return '<html>\n' +
-         '  <head>\n' +
-         '    <style>\n' +
-         '      body {\n' +
-         '        margin: 0;\n' +
-         '        padding: 0;\n' +
-         '        width: 100%;\n' +
-         '        height: 100%;\n' +
-         '        background-repeat: no-repeat;\n' +
-         '        background-size: contain;\n' +
-         '        background-image: url("' + data.url(image) + '");\n' +
-         '      }\n' +
-         '      @media (min-resolution: 2dppx) {\n' +
-         '        body {\n' +
-         '          background-image: url("' + data.url(image.replace(".png", "@2x.png")) + '");\n' +
-         '        }\n' +
-         '      }\n' +
-         '    </style>\n' +
-         '  </head>\n' +
-         '  <body></body>\n' +
-         '</html>\n';
+  return {
+    "16": data.url(image),
+    "32": data.url(image.replace(".png", "@2x.png"))
```

Comment:
The ActionButton doesn't seem to work; the panel doesn't pop up. Switching back to a ToggleButton works though. See my toggle_button branch... https://github.com/gamen/SnoozeTabs/tree/toggle_button